charter.md

```diff
@@ -53,15 +53,207 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Transparency
 
-Steering committee members are generally expected to attend every meeting. We
+The processes and operational model of the committee are intended to sustain a
+healthy open source community.
+Wherever and whenever possible, the committee strives to operate in a
+transparent manner.
+
+In keeping with the community's accepted definition of
+[_"committee"_][committee], committees do not always operate in the open and
+topics may arise that require discretion.
+
+In the absence of a documented process, seated committee members will determine
+an appropriate process and document it for transparency purposes.
+
+[committee]: https://git.k8s.io/community/governance.md#committees
+
+## Changes
+
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members.
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing. As a courtesy,
+such notifications should be given at least 30 calendar days in advance of
+their departure.
+
+The committee can choose to accept a member's resignation and carry out
+offboarding procedures at any time following the written notice.
+
+### Removal
+
+<!--
+
+STEERING CONSIDERATIONS
+
+Removal
+
+- [ ] Special election for specific removal scenarios
+  - What is the perception of having a committee remove a representative that was elected by the community?
+- [ ] Codify the voting process and channels for removal scenarios
+- [ ] Consider CoCC guidance on removals as prior art: https://github.com/kubernetes/community/blob/master/committee-code-of-conduct/charter.md#removal
+- [ ] Consider prior art in changes to maintainership e.g., SPIFFE/SPIRE (https://github.com/spiffe/spire/blob/main/MAINTAINERS.md#changes-in-maintainership)
+
+COMMUNITY CONSIDERATIONS
+
+No Confidence
+
+- [ ] Provide guidelines on "no confidence"
+  - Process transparency could be considered inversely proportional to the importance of these guidelines
+
+Removal
+
+- [ ] Scope of removal of members (one, many)
+- [ ] Precedence for removal scenarios
+- [ ] What details of removal scenarios should be public/private?
+  - Call for removal?
+  - Reason for removal?
+  - Vote on removal?
+  - What are the public artifacts of the removal process?
```

Comment:
If the `Removal` is made a public artifact, it would be good to somehow loop in the project moderators.
I would say the moderators do not need to know every detail but it would be good they be in the loop whether by the the steering committee or the coc committee